src/main/scala/devices/debug/Debug.scala

```diff
@@ -1159,3 +1418,89 @@ class DMIToTL(implicit p: Parameters) extends LazyModule {
 
   }
 }
+
+class SBToTL(implicit p: Parameters) extends LazyModule {
+
+  val node = TLClientNode(Seq(TLClientPortParameters(Seq(TLClientParameters("sba")))))
```

Comment:
what does this name get used for @hcook ? Depending on the answer, it might be better to call it "debug_sba" or similar.

Reply:
I don't think it has impact on any verilog names. It appears when you connect the module to the rest of the diplomacy graph, i.e. `frontBus.fromPort { sb2tl.node }`. I think it's only worth being more specific if a single module has multiple nodes.

Reply:
Changing to "debug_sba" for clarity